odysseus/odysseus_tree/board/tpu/config.txt

```diff
@@ -31,7 +31,7 @@ enable_uart=1
 dtoverlay=krnbt=on
 
 dtoverlay=vc4-kms-v3d-pi4
-dtoverlay=imx219
```

Comment:
lmao why did we have the imx219 overly on. backup cam!

Reply:
it was the default config.  never noticed.